.github/workflows/ci.yml

```
@@ -7,13 +7,15 @@ on:
   pull_request:
     branches:
     - master
+  schedule:
+  - cron: '7 6 * * *'
 
 jobs:
-  # Runs common light-weight checks like linting and type checking.
   api_checks:
     name: API Checks
     runs-on: ubuntu-latest
-
+    # Don't run nightly builds for forks.
```

```suggestion
    # Don't run nightly builds for forks, unless it's a PR to master
```

I'm not sure if that's right, but I think this explains the `github.event_name != 'schedule'` bit.

---

I think it's right, we have the same `if` condition in other repos. We want this to run on PRs and master commits for the main repo and all forks, but we don't want it to run on a nightly schedule for forks (because people have complained about this before).